api/app/controllers/review.controller.ts

```diff
@@ -441,3 +443,37 @@ export const getAllReviewsForAdmin = async (req: Request, res: Response) => {
     }
   }
 };
+
+export const resetReviews = async (req: Request, res: Response) => {
+  const { id } = req.params; // Get the review ID from the request parameters
+
+  try {
+    // Find the review
+    const review = await Review.findOne({ where: { id } });
+
+    if (!review) {
+      return res.status(404).send("Review not found");
+    }
+
+    // Delete the review
+    await Review.destroy({ where: { id } });
+
+    // Reset the transcript
+    await Transcript.update(
+      { status: "queued", claimedBy: null },
+      { where: { id: review.transcriptId } }
+    );
+
+    // Clear the Redis cache
+    redis.del(`review:${id}`, (err, succeeded) => {
```

Comment:
I don't think we're caching reviews

Reply:
You need to also clear the the transcripts cache. 
When testing locally, after claiming a review and then testing the endpoint, I had to also `DEL transcripts:page:2` and `DEL cachedTranscripts` for the transcript to show up again in the page. That said, I am not sure exactly which cache needs to be deleted.